### PR TITLE
Adding checks to helm charts

### PR DESCRIPTION
**Pull request by SrishT**
Signed-off-by: SrishT <Srishti.Thakkar@dell.com>

### Change log description
Adds the necessary checks to the zookeeper charts in order to set values for a subset of the options provided within a given block.

### CODE REVIEW

File: charts/zookeeper/README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ The following table lists the configurable parameters of the Zookeeper chart and
 | `pod.tolerations` | Specifies the pod's tolerations | `[]` |
 | `pod.env` | List of environment variables to set in the container | `[]` |
 | `pod.annotations` | Specifies the annotations to attach to pods | `{}` |
-| `pod.securityContext` | Specifies the security context for the entire pod | |
+| `pod.securityContext` | Specifies the security context for the entire pod | `{}` |
 | `pod.terminationGracePeriodSeconds` | Amount of time given to the pod to shutdown normally | `180` |
```
Comment (anishakj) [inline charts/zookeeper/README.md:70]: Please change termination grace period default value to 30 seconds

**Review by anishakj — CHANGES_REQUESTED** [review]

File: charts/zookeeper/README.md (hunk 1 above)
Comment (SrishT) [reply charts/zookeeper/README.md]: Done

**Review by anishakj — APPROVED** [review]
LGTM

**Review by Prabhaker24 — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ The operator itself is built with the [Operator framework](https://github.com/op
 
 ## Usage
 
-We recommend using our [helm charts](charts) for all installation and upgrades. Since version 0.2.8 onwards, the helm charts for zookeeper operator and zookeeper cluster are published in [https://charts.pravega.io/](https://charts.pravega.io/). To add this repository to your Helm repos, use the following command
+We recommend using our [helm charts](charts) for all installation and upgrades. Since version 0.2.8 onwards, the helm charts for zookeeper operator and zookeeper cluster are published in [https://charts.pravega.io](https://charts.pravega.io/). To add this repository to your Helm repos, use the following command
 ```
```
[the remark this answers is not in the document]
Comment (SrishT) [reply README.md:40]: Rollbacks are supported in the pravega operator and not in the zookeeper operator currently. I will add that note in the Readme in the pravega operator documentation.

**Pull request merged**